docs/contributing/running-and-writing-acceptance-tests.md

```diff
@@ -954,6 +955,45 @@ func testAccDataSourceAwsPricingProductConfigRedshift() string {
 
 If the testing configurations require more than one region, reach out to the maintainers for further assistance.
 
+#### Acceptance Test Concurrency
+
+Certain AWS service APIs allow a limited number of a certain component, while the acceptance testing runs at a default concurrency of twenty tests at a time. For example as of this writing, the SageMaker service only allows one SageMaker Domain per AWS Region. Running the tests with the default concurrency will fail with API errors relating to the component quota being exceeded.
+
+When encountering these types of components, the acceptance testing can be setup to limit the available concurrency of that particular component. When limited to one component at a time, this may also be referred to as serializing the acceptance tests.
+
+To convert to serialized (one test at a time) acceptance testing:
+
+- Convert all existing capital `T` test functions with the limited component to begin with a lowercase `t`, e.g. `TestAccSagemakerDomain_basic` becomes `testAccSagemakerDomain_basic`
```

Comment:
Perhaps we could explain what this change is doing?

```suggestion
- Convert all existing capital `T` test functions with the limited component to begin with a lowercase `t`, e.g. `TestAccSagemakerDomain_basic` becomes `testAccSagemakerDomain_basic`. This will prevent the test framework from executing these tests directly as the prefix `Test` is required.
```

Reply:
Good call, yes!